index.html

```diff
@@ -7,177 +7,352 @@
     <link rel="stylesheet" href="styles.css">
   </head>
 
-  <body>
-    <header>
-      <a id="logo">Mark.O</a>
-      <nav>
-        <img src="./logos/Union.jpeg" alt="menue bar" id="menue-button">
+  
+<body>
+  <div class="hero">
+    <header class="header bg-white">
+      <nav class="navbar flex">
+        <div class="navbar__logo">
+          <a href="#" class="navbar__link link fz-18 fw-700 lh-20">Mark.O</a>
+        </div>
+        <div class="navbar__menu">
+          <ul class="navbar__menu__ul ul flex">
+            <li class="navbar__menu__li">
+              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Portfolio</a>
+            </li>
+            <li class="navbar__menu__li">
+              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">About</a>
+            </li>
+            <li class="navbar__menu__li">
+              <a href="#" class="navbar__menu__link link fw-500 fz-15 lh-20 clr-n-600">Contact</a>
+            </li>
+          </ul>
+        </div>
+        <div class="navbar__hamb">
+          <img src="logos/Union.jpeg" alt="menu icon" />
+        </div>
       </nav>
     </header>
-    <main>
-      <section id="intro">
-        <h1>Hi,Im Mark Otuya</h1>
+  </div>
+
+  
+      <section class="head flex">
+        <h1>Hi,Im <span id="name">Mark Otuya</span></h1>
         <p id="bio">’m a software developer & Web Designer! I can Build your Dream Web Page on Demand,
           Look Below for  some of my work and experience!
            Feel free to contact me, Lets work.</p>
          <h3>Contact Me</h3>
```

Comment:
- [ ] [REQUESTED] kindly as requested by the previous reviewer, fix this issue by adding a margin-top for this  `<h3>` I am sure it would look way better after that.